extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -190,6 +190,14 @@ export class AmpAd3PImpl extends AMP.BaseElement {
     this.boundNoContentHandler_ = () => this.noContentHandler_();
 
     setupA2AListener(this.win);
+
+    /** @private @const {function()|null}*/
+    this.renderStartResolve_ = null;
+
+    /** @private @const {!Promise}*/
+    this.renderStartPromise_ = new Promise(resolve => {
```

Comment:
Yes indeed. But Since we are not using it right now, I think it's better to leave it private for compiler optimization now, and make it public in the PR that use it externally.